[glade] Failed to build

```
[84/248] Generating Gladeui-2.0.gir with a custom command
FAILED: gladeui/Gladeui-2.0.gir 
/usr/local/bin/g-ir-scanner --no-libtool --namespace=Gladeui --nsversion=2.0 --warn-all --output gladeui/Gladeui-2.0.gir --c-include=gladeui/glade.h -I/usr/local/tmp/crew/glade-3.38.2.tar.xz.dir/glade-3.38.2/gladeui -I/usr/local/tmp/crew/glade-3.38.2.tar.xz.dir/glade-3.38.2/build/gladeui -I./. -I../. --filelist=/usr/local/tmp/crew/glade-3.38.2.tar.xz.dir/glade-3.38.2/build/gladeui/libgladeui-2.so.13.0.2.p/Gladeui_2.0_gir_filelist --include=Gtk-3.0 --symbol-prefix=glade --identifier-prefix=Glade --pkg-export=gladeui-2.0 --cflags-begin -DHAVE_CONFIG_H -I./. -I../. -I/usr/local/include/glib-2.0 -I/usr/local/lib/glib-2.0/include -I/usr/local/include/gtk-3.0 -I/usr/local/include/at-spi2-atk/2.0 -I/usr/local/include/at-spi-2.0 -I/usr/local/include/dbus-1.0 -I/usr/local/lib/dbus-1.0/include -I/usr/local/include/gio-unix-2.0 -I/usr/local/include/cairo -I/usr/local/include/pango-1.0 -I/usr/local/include/fribidi -I/usr/local/include/atk-1.0 -I/usr/local/include/pixman-1 -I/usr/local/include/freetype2 -I/usr/local/include/harfbuzz -I/usr/local/include/libdrm -I/usr/local/include/libpng16 -I/usr/local/include/gdk-pixbuf-2.0 -I/usr/local/include/libmount -I/usr/local/include/blkid -I/usr/local/lib/libffi-3.2.1/include -I/usr/local/include/libxml2 -I/usr/local/include/gobject-introspection-1.0 --cflags-end --add-include-path=/usr/local/share/gir-1.0 --library gladeui-2 -L/usr/local/tmp/crew/glade-3.38.2.tar.xz.dir/glade-3.38.2/build/gladeui -lglib-2.0 -lgmodule-2.0 -lgtk-3 -lgdk-3 -lz -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lxml2 -lm -lgirepository-1.0
Traceback (most recent call last):
  File "/usr/local/bin/g-ir-scanner", line 98, in <module>
    from giscanner.scannermain import scanner_main
  File "/usr/local/lib/gobject-introspection/giscanner/scannermain.py", line 35, in <module>
    from giscanner.ast import Include, Namespace
  File "/usr/local/lib/gobject-introspection/giscanner/ast.py", line 29, in <module>
    from .sourcescanner import CTYPE_TYPEDEF, CSYMBOL_TYPE_TYPEDEF
  File "/usr/local/lib/gobject-introspection/giscanner/sourcescanner.py", line 35, in <module>
    from giscanner._giscanner import SourceScanner as CSourceScanner
ModuleNotFoundError: No module named 'giscanner._giscanner'
[85/248] Generating symbol file gladeui/libgladeui-2.so.13.0.2.p/libgladeui-2.so.13.0.2.symbols
[86/248] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-activatable-editor.c.o
[87/248] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-accels.c.o
[88/248] Compiling C object src/glade.p/glade-window.c.o
[89/248] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-app-chooser-button-editor.c.o
ninja: build stopped: subcommand failed.
glade failed to build: `ninja -C build` exited with 1
```

--- a/packages/glade.rb
+++ b/packages/glade.rb
@@ -11,6 +11,7 @@ class Glade < Package
   depends_on 'libxml2'
   depends_on 'gobject_introspection'
   depends_on 'pygobject'
+  depends_on 'graphite'
 
   def self.build
     ENV['CFLAGS'] = "-fuse-ld=lld"

